src/package-change-checker.spec.js

```diff
@@ -200,5 +200,24 @@ describe('package-change-checker.js', () => {
         expect(actual).to.be.true();
       });
     });
+
+    describe('with post-checkout hook providing two hashes', () => {
+      before(() => {
+        checker.hasChangedDependencies([
+          'abcdef0123456789abcdef0123456789abcdef01',
+          'abcdef0123456789abcdef0123456789abcdef02',
+        ]);
+      });
+
+      after(teardown);
+
+      it('should get the diff with the two provided hashes', () => {
```

Comment:
Could you also add a test to cover the passing of the commit hashes into the `fileContentLoader` functions?